lib/dashboard_page.py

```diff
@@ -0,0 +1,25 @@
+"""DashboardPage"""
+from dataclasses import dataclass
+from typing import Callable
+from lib.url import selected_filters
+
+
+@dataclass
+class DashboardPage:
```

Comment:
Sorry to be a pain and only bring this up now. But could we move this into our package? These lib functions seem generic enough now for us to be able to do so?